### PR TITLE
Cleaned up some .tf files

### PR DESCRIPTION
**Pull request by dginty4**

### CODE REVIEW

File: docs/resources/routing_email_domain.md [hunk 1]
```diff
@@ -20,8 +20,8 @@ The following Genesys Cloud APIs are used by this resource. Ensure your OAuth Cl
 ## Example Usage
 
 ```terraform
-resource "genesyscloud_routing_email_domain" "test-example-com" {
-  domain_id             = "test.example.com"
+resource "genesyscloud_routing_email_domain" "example-example-com" {
+  domain_id             = "example.example.com"
```
Comment (charliecon) [inline docs/resources/routing_email_domain.md:24]: @dginty4  `example.domain.com` might look better here, I think. Same for the resource ID on the line above.

File: docs/resources/routing_email_route.md [hunk 2]
```diff
@@ -21,23 +21,23 @@ The following Genesys Cloud APIs are used by this resource. Ensure your OAuth Cl
 
 ```terraform
 resource "genesyscloud_routing_email_route" "support-route" {
-  domain_id    = "test.example.com"
+  domain_id    = "example.example.com"
```
Comment (charliecon) [inline docs/resources/routing_email_route.md:24]: Same here

File: docs/resources/routing_email_route.md [hunk 3]
```diff
@@ -21,23 +21,23 @@ The following Genesys Cloud APIs are used by this resource. Ensure your OAuth Cl
...
   pattern      = "support"
-  from_name    = "Test Support"
-  from_email   = "testsupport@test.example.com"
-  queue_id     = genesyscloud_routing_queue.test-queue.id
+  from_name    = "Example Support"
+  from_email   = "examplesupport@example.example.com"
```
Comment (charliecon) [inline docs/resources/routing_email_route.md:27]: , and here. And for the other two instances below

**Review by ronanwatkins — APPROVED** [review]

**Comment by ronanwatkins** [comment]
Very good. Our docs will look a lot cleaner now

File: examples/data-sources/genesyscloud_user/data-source.tf [hunk 4]
```diff
@@ -1,3 +1,3 @@
 data "genesyscloud_user" "user" {
-  email = "test@example.com"
+  email = "sample@example.com"
```
Comment (charliecon) [inline examples/data-sources/genesyscloud_user/data-source.tf:2]: `user@example.com` I think would be better.

File: examples/resources/genesyscloud_routing_email_route/resource.tf [hunk 5]
```diff
@@ -1,20 +1,20 @@
 resource "genesyscloud_routing_email_route" "support-route" {
-  domain_id    = "test.example.com"
+  domain_id    = "example.example.com"
```
Comment (charliecon) [inline examples/resources/genesyscloud_routing_email_route/resource.tf:2]: Same for this resource. `example.example.com` -> `example.domain.com`

File: examples/resources/genesyscloud_routing_queue/resource.tf [hunk 6]
```diff
@@ -1,7 +1,7 @@
-resource "genesyscloud_routing_queue" "test_queue" {
-  name                              = "Test Queue"
+resource "genesyscloud_routing_queue" "example_queue" {
+  name                              = "Example Queue"
   division_id                       = genesyscloud_auth_division.home.id
-  description                       = "This is a test queue"
+  description                       = "This is a example queue"
```
Comment (charliecon) [inline examples/resources/genesyscloud_routing_queue/resource.tf:4]: a -> an 
Or just "Example description", like it is in some other resources.

**Comment by charliecon** [comment]
@ronanwatkins Apologies, didn't realised this was closed. I must have been reading through while you merged.

**Comment by ronanwatkins** [comment]
> @ronanwatkins Apologies, didn't realised this was closed. I must have been reading through while you merged.

My bad, I should have waited for more reviews. @dginty4 Could you open a new PR to address Charlie's comments?

**Pull request merged**